.github/workflows/main.yml

```diff
@@ -40,11 +45,21 @@ jobs:
           rm -rf example_docs/_build_legacy
       - name: Build Furo theme
         run: |
-          THEME=_qiskit_furo BUILD_DIR=example_docs/_build_furo/html tox run -e docs
-          tar -zcvf furo_html_docs.tar.gz example_docs/_build_furo/html
+          THEME=_qiskit_furo tox run -e docs
+          tar -zcvf furo_html_docs.tar.gz example_docs/docs/_build/html
           mv furo_html_docs.tar.gz artifacts/.
-          rm -rf example_docs/_build_furo
-      - uses: actions/upload-artifact@v2
+      - name: Upload Sphinx builds
+        uses: actions/upload-artifact@v3
+        if: always()
         with:
           name: html_docs
           path: artifacts
+
+      - name: Run JavaScript and snapshot tests
+        run: npm run _run-tests
```

Comment:
Note that this isn't the standard `npm test`, which will use Docker. I optimized `npm test` to be the ideal experience for local developers.